entries/a/autocode.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Autocode": {
+    "domain": "autocode.com",
+    "img": "autocode.com.png",
```

Comment:
SVG-files are prefered over PNGs. I suggest you use this one:
![autocode.com.svg](https://camo.githubusercontent.com/3a9ac2ab1d87af8aab1cfcb37413a81fa8b2ba2b72d6ef5f5d853e68c4083259/68747470733a2f2f642e74656d7066696c65732e646f776e6c6f61642f44363236393937373539393137372f3f703d354437396a70)
```suggestion
```

Reply:
Oh thanks, I wasn't able to find an svg.